extensions/indexes/lucene/src/org/exist/indexing/lucene/AnalyzerConfig.java

```diff
@@ -250,34 +252,34 @@ protected static Analyzer configureAnalyzer(Element config) throws DatabaseConfi
     private static Analyzer createInstance(final Class<?> clazz, final Class<?>[] vcParamClasses,
         final Object[] vcParamValues, final boolean warnOnError) {
 
-        String className = clazz.getName();
+        final String className = clazz.getName();
 
-        try {
-            final Constructor<?> cstr = clazz.getDeclaredConstructor(vcParamClasses);
-            cstr.setAccessible(true);
+        MethodType constructorType = methodType(void.class);
```

Comment:
can this one be  made static final too?

Reply:
It can change just two lines below! `MethodType` is immutable, so it has to be reassigned to the new reference when a new instance is created.

Reply:
sorry missed that